### DISCUSSION TITLE
phoebe program version 0.32

### DISCUSSION THREAD

**Discussion by helenkurd** [phoebe-project/phoebe1#16] [General]
hello i have a question in phoebe program version 0.32 how I fit light curve for a binary star to determine initial parameters

**Comment by kecnry** [comment]
The [documentation for phoebe legacy/1.0/0.32](http://phoebe-project.org/1.0/docs) should be a good resource to get started.  If you have any specific questions, let us know and we'd be happy to try to help.  Do note that that version of PHOEBE is no longer under active development, so you may want to consider using PHOEBE 2 which now comes with built-in wrappers around many estimators, optimizers, and samplers to find the solution.  Although it has a bit of a learning curve, the [examples and tutorials for PHOEBE 2](http://phoebe-project.org/docs) are quite extensive.

(I'll also move this to the discussion board on the phoebe1 repository to avoid any confusion)

**Reply by helenkurd** [reply]
thanks ver much sir

**Reply by helenkurd** [reply]
sir this version of phoebe 2 can fit light curve for a binary star and determine initial parameter such as (inclination, mass ratio , surface potential, tempreature, gravity darkening, reflection)

**Reply by kecnry** [reply]
yes, see section 8 (and particularly the LC and RV estimators) in the [online tutorials](http://phoebe-project.org/docs/latest/tutorials).  There still is manual tweaking required, but the estimators attempt to get reasonable starting values from measuring the geometry of your observables.

**Reply by helenkurd** [reply]
thanks very much sir

**Comment by helenkurd** [comment]
hello sir is there any pdf manual to learning how to use phoebe 2.3

**Reply by kecnry** [reply]
No, all the documentation is online at the link above. Each tutorial is a Jupiter notebook, so you can download those or clone the entire repo if you need offline access.

**Comment by helenkurd** [comment]
thanks sir but sir i have data in excel file how ican enter to phobe 2.3
and plotting it i use this program at first time and no one know this
program here please sir help me to learn i am msc student thanks sir

**Reply by kecnry** [reply]
Excel files are not natively supported, so you will need to parse the file yourself into arrays (using something like [np.loadtxt](https://numpy.org/doc/stable/reference/generated/numpy.loadtxt.html)) and pass it to phoebe following the tutorials @gecheline posted in the [other thread](https://github.com/phoebe-project/phoebe2/discussions/495#discussioncomment-1484884).

My suggestion would be to read through those tutorials and try to adapt them to your case.  If you get stuck or something you need isn't covered in the tutorials, start a new thread in the [phoebe2 discussion board](https://github.com/phoebe-project/phoebe2/discussions/new), showing the script/notebook you have and where you're stuck.
